dir_files.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# This script count all files of a given type in the specified path
+#
+# @author Wojciech Lesniewski
+#
+DIRECTORY=$1
+
+if [ $# -eq 0 ]; then
+  echo "Usage /path/to/script directory"
+  exit 3
+elif [ ! -d "$DIRECTORY" ]; then
+  echo "ERROR: /path/to/dir/ is not directory"
```

Comment:
This is cool, thank you. Maybe it would be good to change `/path/to/dir` in the message with the actual directory